applications/processor/script-processor/src/test/java/org/springframework/cloud/stream/app/processor/script/ScriptProcessorIntegrationTests.java

```diff
@@ -44,7 +43,6 @@
  * @author Chris Schaefer
  * @author Soby Chacko
  */
-@Disabled
```

Comment:
@sobychacko , what was wrong with this test?
Doesn't look like it fails somehow: I really just removed this annotation and it passes.

Thanks

Reply:
It failed for me and I believe it was related to compatibility matrix (which was old and requiring boot 2.4,2.5). I would be sure to re-run `./setup-build.sh` to make sure it is using the versions in the poms. I definitely saw all of these tests fail before I "fixed" them.